semantic_conventions/resource/engine.yaml

```
@@ -0,0 +1,22 @@
+groups:
+  - id: engine_resource
+    prefix: engine
+    brief: >
+       Resource describing the packaged software running the application code. Engines are executed using process.runtime.
```

```suggestion
       Resource describing the packaged software running the application code. Engines are typically executed using process.runtime.
```
see https://github.com/open-telemetry/opentelemetry-specification/pull/1293/files#r567805983